doc/source/configuration/wazuh.rst

```diff
@@ -254,6 +254,33 @@ It will be used by wazuh secrets playbook to generate wazuh secrets vault file.
   kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/wazuh-secrets.yml
   ansible-vault encrypt --vault-password-file ~/vault.pass $KAYOBE_CONFIG_PATH/wazuh-secrets.yml
 
+Configure Wazuh Dashboard's Server Host 
+---------------------------------------
+
+By default, the Dashboard will be hosted on a random IP. It is best practice to manually set the dashboard's server host.
```

Comment:
Very nitpicky, but I don't like the phrase "a random IP". Can we just drop this sentence?
```suggestion
It is best practice to manually set the dashboard's server host.
```